.env.example

```diff
@@ -24,4 +24,4 @@ MAIL_HOST=mailtrap.io
 MAIL_PORT=2525
 MAIL_USERNAME=null
 MAIL_PASSWORD=null
-MAIL_ENCRYPTION=null
+MAIL_ENCRYPTION=null
```

Comment:
ah yes! I thought I had the package that it did that automatically but looks like I'll have to re-install that - thank you!